src/main/java/org/ambraproject/rhino/config/RuntimeConfiguration.java

```diff
@@ -177,4 +177,9 @@ public String getConfigKey() {
    */
   String getManuscriptCustomMetaName(ManuscriptCustomMetaAttribute attribute);
 
+  /**
+   * @return the set of enabled dev features, configured in wombat.yaml.
+   */
+  ImmutableSet<String> getEnabledDevFeatures();
```

Comment:
(nit) usually prefer that a method returns a Java interface (i.e. `Set`) rather than an implementing class so that the code is not tied to a specific library.

Reply:
This code was coped/pasted from wombat verbatim, but I agree with your opinion.